pinot-common/src/main/java/org/apache/pinot/common/function/TransformFunctionType.java

```diff
@@ -90,6 +90,7 @@ public enum TransformFunctionType {
   ARRAYMIN("arrayMin"),
   ARRAYMAX("arrayMax"),
   ARRAYSUM("arraySum"),
+  ARRAYCARDINALITY("arrayCardinality", "cardinality"),
```

Comment:
Why are we overloading arrayCardinality with arrayLength? Are they the same?

Reply:
yes they are the same. technically speaking we should create an "CardinalityTransformOperator" and should accept any input (map/list/mv/sv) and 
- map will be map.size()
- list will be list.size()
- mv will be mv.length
- sv will always return literal 1

but since CARDINALITY should only work against mv/list/map. it is currently equivalent to ARRAYLENGTH

Reply:
In that case we should just add `cardinality` as an alias of `ARRAYLENGTH`?

Reply:
good point.